src/typescript/13-utility-types/utility-types.md

```diff
@@ -1,17 +1,17 @@
 @page learn-typescript/utility-types Utility Types
 @parent learn-typescript 13
+@outline 3
 
 @description Learn About the Utility Types provided by TypeScript
 
 @body
 
 ## Overview
 
-Utility Types are built-in helper Types meant for common Type transformations.
+In this section, you will:
 
-In this section you will see a small description for the most used Utility Types.
-
-We are going to repeat the interface Dinosaur below throughout the examples.
+- Review popular utility types
```

Comment:
Getting writers block in the overview, is this enough?

Reply:
The points under this section should match the concept headers. In this case, I think we could make them better by writing something like:

- Make properties optional with `Partial<Type>`
- Make properties required with `Required<Type>`
- Set properties as immutable with `Readonly<Type>`
- Map keys to a type with `Record<Keys, Type>`
- Include specific properties with `Pick<Type, Keys>`
- Exclude specific properties with `Omit<Type, Keys>`
- Exclude values from a type with `Exclude<Type, ExcludedUnion>`
- Select included subtypes with `Extract<Type, Union>`
- Remove nullability with `NonNullable<Type>`
- Get function return type with `ReturnType<Type>`